src/main/resources/com/mathworks/ci/RunMatlabBuildBuilder/help-buildOptions.html

```diff
@@ -0,0 +1,7 @@
+<div>
+  Specify a space-separated list of MATLAB build options. The plugin supports
+  the same build options that you can pass to the buildtool command when running
```

Comment:
Shouldn't buildtool be bolded here? <b>buildtool</b>

Reply:
Updated!